chainerx/__init__.pyi

```diff
@@ -314,6 +314,10 @@ class ndarray:
 
     def __neg__(self) -> ndarray: ...
 
+    def __or__(self, arg0: tp.Any) -> ndarray: ...
+
+    def __pow__(self, arg0: tp.Any) -> ndarray: ...
+
```

Comment:
- ChainerX's `__pow__` is from v7.
- `__or__` looks irrelevant.

Reply:
Thanks. It maybe needs to just removes `square` declaration in this backport.